src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -263,7 +263,31 @@ private <T extends SyntaxElement> T parse(Iterator<? extends SyntaxElementInfo<?
 								x = x2;
 							}
 							T t = info.c.newInstance();
-							if (t.init(res.exprs, i, getParser().getHasDelayBefore(), res)) {
+							ParserInstance parser = getParser();
+							boolean error = false;
```

Comment:
Replace the error boolean with `continue`

Reply:
Oh yeah this should be possible

Reply:
what do you mean sorry

Reply:
Instead of having a boolean to check if the code errored, you could use `continue` when you error so it goes straight to the next element without proceeding with the rest of the code

Reply:
oh wait i completely missed that, i'll get on that in a minute